Production_scripts/B01_acute_draft.R

```diff
@@ -0,0 +1,249 @@
+    costsfy = `Costs_Financial_Year_(01)`,
+    diag1 = `Diagnosis_1_Code_(6_char)`,
+    diag2 = `Diagnosis_2_Code_(6_char)`,
+    diag3 = `Diagnosis_3_Code_(6_char)`,
+    diag4 = `Diagnosis_4_Code_(6_char)`,
+    diag5 = `Diagnosis_5_Code_(6_char)`,
+    diag6 = `Diagnosis_6_Code_(6_char)`,
+    dischto = Discharge_Trans_To_Code,
+    disch = Discharge_Type_Code,
+    falls_adm = `Falls_Related_Admission_(01)`,
+    lca = Geo_Council_Area_Code,
+    DataZone = Geo_Data_Zone_2011,
+    postcode = `Geo_Postcode_[C]`,
+    HSCP = `Geo_HSCP_of_Residence_Code_-_current`,
+    conc = `Lead_Consultant/HCP_Code`,
+    admloc = Location_Admitted_Trans_From_Code,
+    dischloc = Location_Discharged_Trans_To_Code,
+    mpat = Management_of_Patient_Code,
+    hbrescode = `NHS_Board_of_Residence_Code_-_current`,
+    nhshosp = `NHS_Hospital_Flag_(01)`,
+    yearstay = `Occupied_Bed_Days_(01)`,
+    oldtadm = Old_SMR1_Type_of_Admission_Code,
+    op1a = `Operation_1A_Code_(4_char)`,
+    op1b = `Operation_1B_Code_(4_char)`,
+    op2a = `Operation_2A_Code_(4_char)`,
+    op2b = `Operation_2B_Code_(4_char)`,
+    op3a = `Operation_3A_Code_(4_char)`,
+    op3b = `Operation_3B_Code_(4_char)`,
+    op4a = `Operation_4A_Code_(4_char)`,
+    op4b = `Operation_4B_Code_(4_char)`,
+    gender = Pat_Gender_Code,
+    chi = Pat_UPI,
+    cat = Patient_Category_Code,
+    gpprac = Practice_Location_Code,
+    hbpraccode = `Practice_NHS_Board_Code_-_current`,
+    selfharm_adm = `Self_Harm_Related_Admission_(01)`,
+    sigfac = Significant_Facility_Code,
+    spec = `Specialty_Classificat._1/4/97_Code`,
+    submis_adm = `Substance_Misuse_Related_Admission_(01)`,
+    cost_total_net = `Total_Net_Costs_(01)`,
+    location = Treatment_Location_Code,
+    hbtreatcode = `Treatment_NHS_Board_Code_-_current`,
+    uri = Unique_Record_Identifier,
+    record_keydate1 = `Date_of_Admission(01)`,
+    record_keydate2 = `Date_of_Discharge(01)`,
+    dateop1 = `Date_of_Operation_1_(01)`,
+    dateop2 = `Date_of_Operation_2_(01)`,
+    dateop3 = `Date_of_Operation_3_(01)`,
+    dateop4 = `Date_of_Operation_4_(01)`,
+    dob = `Pat_Date_Of_Birth_[C]`,
+    ipdc = Inpatient_Day_Case_Identifier_Code,
+    cij_ipdc = `CIJ_Inpatient_Day_Case_Identifier_Code_(01)`,
+    lineno = `Line_Number_(01)`
+  )
+
+
+# Data Cleaning ---------------------------------------
+
+acute_clean <- acute_file %>%
+  # Set year variable
+  mutate(year = year) %>%
+  # Set recid as 01B and flag GLS records
+  mutate(recid = if_else(GLS_Record == "Y", "GLS", "01B")) %>%
+  # Set IDPC marker for the episode
+  mutate(ipdc = case_when(
+    ipdc == "IP" ~ "I",
+    ipdc == "DC" ~ "D"
+  )) %>%
+  # Set IDPC marker for the cij
+  mutate(cij_ipdc = case_when(
+    cij_ipdc == "IP" ~ "I",
+    cij_ipdc == "DC" ~ "D"
+  )) %>%
+  # Recode GP practice into 5 digit number
+  # We assume that if it starts with a letter it's an English practice and so recode to 99995.
+  eng_gp_to_dummy(gpprac) %>%
```

Comment:
Function has been renamed
```suggestion
  convert_eng_gpprac_to_dummy(gpprac) %>%
```